docs/src/intro.md

```diff
@@ -1,6 +1,7 @@
 # bootc
 
 Transactional, in-place operating system updates using OCI/Docker container images.
+bootc is the key component in a broader mission of [bootable containers](https://containers.github.io/bootable/).
```

Comment:
Very minor nit, but it's a bit awkward to start the sentence with bootc as a lower-case 'b', but big 'B' seems equally wrong.  A quick scan elsewhere in the docs turned up https://containers.github.io/bootc/building/guidance.html#systemd-units which has a sentence beginning with "The bootc project".  I suspect maybe @cgwalters was trying to avoid the same thing.

Reply:
Yeah agree, but...not worth a respin for this I think right now.